projects/step-frontend/src/lib/modules/timeseries/performance-view/filter-bar/filter-bar.component.ts

```diff
@@ -184,18 +176,18 @@ export class FilterBarComponent implements OnInit, OnDestroy {
   }
 
   handleFilterChange(index: number, item: TsFilterItem) {
-    this.visibleFilters[index] = item;
+    this.filterItems[index] = item;
     this.emitFilterChange$.next();
   }
 
   addFilterItem(item: TsFilterItem) {
-    const filterIndex = this.visibleFilters.findIndex((i) => i.attributeName === item.attributeName);
+    const filterIndex = this.filterItems.findIndex((i) => i.attributeName === item.attributeName);
 
     if (filterIndex !== -1) {
       const index =
-        filterIndex < this.visibleFilters.length - this.visibleFilters.length
+        filterIndex < this.filterItems.length - this.filterItems.length
           ? filterIndex
-          : filterIndex + this.visibleFilters.length;
+          : filterIndex + this.filterItems.length;
```

Comment:
Always put calculations in parantheses to make it clear in which order they are compared / calculated

Reply:
I think the parentheses are getting removed by Prettier as they are not technically needed